Add Thieves' Night On the Town -- Tnott

megapatato opened the pull request:
[Thieves Guild Requirements](https://www.nexusmods.com/skyrimspecialedition/mods/33256) and [Thieves' Night on the Town](https://www.nexusmods.com/skyrimspecialedition/mods/42069)

Both plugins declare entries for MarkarthNepossHouse "Nepos's House" [CELL:00016DFB]

Thieves Guild Requirements adds a persistent reference. Thieves Night on the Town changes the cell's flags and locklist.

Both modifications play fine together, but loading TGR after Tnott will undo Tnott's changes; loading Tnott after TGR will preserve the changes from both.

If submission needs style cleaning / adjustment, please advise.

megapatato committed:
Add sorting rule for Thieves Guild Requirements and Thieves' Night on the Town

Both plugins declare entries for MarkarthNepossHouse "Nepos's House" [CELL:00016DFB]

Thieves Guild Requirements adds a persistent reference.
Thieves Night on the Town changes the cell's flags and locklist.

Both modifications play fine together, but loading TGR after Tnott will undo Tnott's changes; loading Tnott after TGR will preserve the changes from both.

sibir-ine commented on masterlist.yaml:
```diff
@@ -29006,3 +29006,9 @@ plugins:
+
+  - name: 'mpThievesNightOnTheTown.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42069'
+        name: "Thieves' Night On the Town -- Tnott"
```
The link just needs a trailing slash, & we prefer to escape single quotes in strings rather than use double quotes:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42069/'
        name: 'Thieves'' Night On the Town -- Tnott'
```

megapatato committed:
Accept style changes, add bash tags

* C.Locklist, as MarkarthNepossHouse cell changes from being a single NPC, to a formlist of residents
* C.MiscFlags, as MarkarthNepossHouse cell is not longer flagged as a public area
* C.Owner, to reset ownership of MorthalThaumaturgistHouse and prevent locking issues with the new NPC
* Keywords, as many Locations gained the keywords TGRWealthyHome and TGBusiness
* Names, as a couple Locations had their FULL - Name record changed to match the name of the cell they contain, instead of a non-existant one.

megapatato commented:
The style suggestion was accepted. Five BashTags were also added.

If there's additional adjustments, please advise.

sibir-ine commented on masterlist.yaml:
```diff
@@ -29006,3 +29006,16 @@ plugins:
+    after: [ 'Thieves Guild Requirements.esp' ]
+    tag:
+      - C.Locklist
+      - C.MiscFlags
```
The `DATA/Public Area` flag is covered by **C.Owner**:

```suggestion
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -29006,3 +29006,16 @@ plugins:
+      - C.Owner
+      - Keywords
+      - Names
+
```
The masterlist only needs one extra newline:

```suggestion
```

megapatato committed:
Accept suggestions

- Remove C.MiscFlags as its intended change should already be covered by usage of C.Owner tag
- Remove extra newline at end of file

megapatato commented:
The suggestions are now included.

I tried accepting the suggestions directly on this PR thread, but after clicking "Commit suggestion", I would get some error about the diff being updated and requiring a refresh, but no commit would have been added. So I manually committed on my local repo.

sibir-ine reviewed (approved):
Yeah, it happens sometimes, not sure why. Looks good though, thanks!
